skfda/representation/_functional_data.py

```diff
@@ -85,6 +86,18 @@ def ndim_codomain(self):
         """
         return self.ndim_image
 
+    @property
+    @abstractmethod
+    def coordinate(self):
```

Comment:
I would use the plural form `coordinates`.
```suggestion
    def coordinates(self):
```

Reply:
Changed in fa02b84.